src/lib/jsonParse/jsonAppendContextElementRequest.cpp

```diff
@@ -60,7 +60,7 @@ static std::string contextAttribute(const std::string& path, const std::string&
 {
   LM_T(LmtParse, ("Got an attribute"));
   reqData->acer.attributeP = new ContextAttribute();
-  reqData->acer.attributeP->valueType = orion::ValueTypeNone;
+  reqData->acer.attributeP->valueType = orion::ValueTypeNotGiven;
```

Comment:
Perhaps we should put `valueType = orion::ValueTypeNotGiven` in the constructor(s) of `ContextAttribute`?

Reply:
Fixed in 707138e0b5ab6fc8c4642afa906290b48fd3f7ae

Not only the ContextAttribute constructor, but also Metatada constructor, with additional fixes in the metadata parsing logic suprisingly unnoticed until now. I'd suggest to review that commit .